Source/Core/Core/PowerPC/JitCommon/JitAsmCommon.h

```diff
@@ -40,6 +40,12 @@ class CommonAsmRoutinesBase
   const u8** pairedLoadQuantized;
 
   // In: array index: GQR to use.
+  // In: ECX: Address to read from.
+  // Out: XMM0: Bottom 32-bit slot holds the read value.
+  // Trashes: all three RSCRATCH
+  const u8** singleLoadQuantized;
```

Comment:
Not too sure, but is this a place where `std::array` could help us get rid of some magic constants (such as the `8` in `8 * sizeof(u8*)` for `ReserveCodeSpace` inside `CommonAsmRoutines::GenQuantizedXxxLoads`)?

Then again, this would probably also apply to the paired ones above, so it could be out of scope here and should be its own PR...